extensions/common/gcp/gcp-core/src/main/java/org/eclipse/edc/gcp/bigquery/service/BigQueryFactoryImpl.java

```diff
@@ -25,36 +23,22 @@
 import org.eclipse.edc.spi.monitor.Monitor;
 
 import java.io.IOException;
-import java.util.Arrays;
 
 public class BigQueryFactoryImpl implements BigQueryFactory {
     private final GcpConfiguration gcpConfiguration;
     private final Monitor monitor;
+    private final IamService iamService;
 
-    public BigQueryFactoryImpl(GcpConfiguration gcpConfiguration, Monitor monitor) {
+    public BigQueryFactoryImpl(GcpConfiguration gcpConfiguration, Monitor monitor, IamService iamService) {
         this.gcpConfiguration = gcpConfiguration;
         this.monitor = monitor;
+        this.iamService = iamService;
     }
 
     @Override
     public BigQuery createBigQuery(GcpServiceAccount serviceAccount) throws IOException {
```

Comment:
no `IOException` is thrown by this method

Reply:
Done